packages/loctool/test/ResourceConvert.test.js

```diff
@@ -288,110 +271,137 @@ describe("resource conversion functions", function() {
         expect(string.getComment()).toBe("no comment");
         expect(string.getState()).toBe("new");
     });
+});
 
-    test("convert string target to plural", function() {
-        expect.assertions(2);
-
-        var string = new ResourceString({
+describe("convertICUToPluralRes", () => {
+    test("converts string target to plural", function() {
+        const string = new ResourceString({
             sourceLocale: "en-US",
             source: "{count, plural, one {There is {n} string.} other {There are {n} strings.}}",
             targetLocale: "de-DE",
             target: "{count, plural,  one {Es gibt {n} Zeichenfolge.} other {Es gibt {n} Zeichenfolgen.}}",
         });
-
-        var expected = {
+        const expected = {
             one: "Es gibt {n} Zeichenfolge.",
             other: "Es gibt {n} Zeichenfolgen.",
         };
-        var plural = conv.convertICUToPluralRes(string);
+
+        const  plural = convert.convertICUToPluralRes(string);
+
+        expect(plural.getType()).toBe("plural");
+        expect(plural.getTargetPlurals()).toStrictEqual(expected);
+    });
+
+    test("converts string target to plural when it contains HTML tags", function() {
+        const string = new ResourceString({
+            sourceLocale: "en-US",
+            source: "{count, plural, one {There is {n} <b>string</b>.} other {There are {n} <b>strings</b>.}}",
+            targetLocale: "de-DE",
+            target: "{count, plural,  one {Es gibt {n} <b>Zeichenfolge</b>.} other {Es gibt {n} <b>Zeichenfolgen</b>.}}",
+        });
+        const expected = {
+            one: "Es gibt {n} <b>Zeichenfolge</b>.",
+            other: "Es gibt {n} <b>Zeichenfolgen</b>.",
+        };
+
+        const plural = convert.convertICUToPluralRes(string);
+
         expect(plural.getType()).toBe("plural");
         expect(plural.getTargetPlurals()).toStrictEqual(expected);
     });
 
-    test("convert string source to plural when the target has more plural categories than the source", function() {
-        expect.assertions(2);
+    test("converts string target to plural when it contains nested HTML tags", function() {
+        const string = new ResourceString({
+            sourceLocale: "en-US",
+            source: "{count, plural, one {There is {n} <span><b>string</b></span>.} other {There are {n} <span><b>strings</b></span>.}}",
```

Comment:
Maybe it would be nice to add tests for reconstruction of
* self-closing tag (`<a />` and/or `<a/>`)
* opening tag with no matching close (`<a>`)
* open-close pair with no children (`<a></a>`)

since ATM you've added
```js
return `<${node.value}>${children}</${node.value}>`;
```
which I think should be fine because IIRC icu-messageparser
* for self-closing it does not produce tag node (i.e. treats it as plain text)
* for unclosed tag it throws an error
but it's not obvious behaviour

Reply:
Added suggested test cases.

https://github.com/iLib-js/ilib-mono/pull/28/commits/5cd22a56b8f47f0f3692cfcca039c8616350b9a8